src/client/components/MenuDropdownGrid/MenuDropdownGrid.react.js

```diff
@@ -14,49 +12,41 @@ const propTypes = {
   items: Types.arrayOf(Types.shape({
     svg: Types.string,
     label: Types.string
-  }))
+  })),
+  setIsHidden: Types.func
```

Comment:
> Related to all changes in this PR to hide applications menu icon.

**Current applications menu configuration**

```jsx
 <MenuIcon
        onClick={() => console.log('click!')}
        svg={_scope.getIcon('apps')}
        title="Applications"
        hideDropdownArrow={true}
      >
        <MenuDropdownGrid
          items={[
            { label: 'Analytics', svg: _scope.getIcon('app_analytics'), onClick: () => console.log('analytics click'), active: true },
            { label: 'Business Network Portal', svg: _scope.getIcon('app_business_network_portal'), href: 'https://opuscapita.com' },
            { label: 'Catalog Portal', svg: _scope.getIcon('app_catalog_portal'), disabled: true },
            { label: 'Contracts', svg: _scope.getIcon('app_contracts') },
            { label: 'Document Portal', svg: _scope.getIcon('app_document_portal') },
            { label: 'Inventory', svg: _scope.getIcon('app_inventory') },
            { label: 'Invoice', svg: _scope.getIcon('app_invoice'), href: 'https://opuscapita.com' },
            { label: 'Liquidity', svg: _scope.getIcon('app_liquidity'), href: 'https://opuscapita.com' },
            { label: 'Order', svg: _scope.getIcon('app_order'), href: 'https://opuscapita.com', disabled: true },
            { label: 'Payments', svg: _scope.getIcon('app_payments') },
            { label: 'Product Information', svg: _scope.getIcon('app_product_information'), href: 'https://opuscapita.com' },
            { label: 'Receivables Collection', svg: _scope.getIcon('app_receivables_collection'), href: 'https://opuscapita.com' },
            { label: 'RFQ', svg: _scope.getIcon('app_rfq'), href: 'https://opuscapita.com' },
            { label: 'Shop', svg: _scope.getIcon('app_shop'), href: 'https://opuscapita.com', disabled: true },
            null,
            { label: 'Supplier Information', svg: _scope.getIcon('app_supplier_information'), href: 'https://opuscapita.com' }
          ]}
        />
      </MenuIcon>
```

**Proposal**

Instead of adding additional state and callbacks to `MenuIcon` and `MenuDropdownGrid` components, maybe its better to add a new component `ApplicationsMenuIcon`.

But show

```jsx
<ApplicationsMenuIcon
   onClick={() => console.log('click!')}
   title="Applications"
   items={[ ... ]}
/>
```



```jsx
// ApplicationsMenuIcon.react.js
...
render() {
	if (!this.props.items.length) {
		return null;
	}
	...
	return (
       // Same components as before changes
	);
}
```